dcsc_setups/PendulumActuators actuators
---
bool success
string message
